packer/ansible/roles/monorail/files/auth_config/config.json

```diff
@@ -0,0 +1,65 @@
+{
+    "CIDRNet": "172.31.128.0/22",
```

Comment:
@WangWinson  Since this configuration is backwards compatible I would just overwrite the files/config.json with the new config.

@heckj  fyi once this goes in we'll need to generate a new vagrant box and update Atlas.